frontend/src/localization/de.json

```diff
@@ -95,12 +95,12 @@
     "description": "Beschreibung",
     "noDescriptionProvided": "Keine Beschreibung verfügbar.",
     "selectAFilter": "Kein Filter ausgewählt.",
-    "dropConcept": "Füge weitere Konzepte des selben Konzeptbaumes hinzu",
+    "dropConcept": "Füge weitere Konzepte des selben Konzeptes hinzu",
```

Comment:
Hmm, now we'll need another word for the concept root like "des selben Root-Konzeptes?" or "Füge weitere Kind-Konzepte des selben Konzeptes hinzu" ?

Reply:
Ah or like below "Konzept-Wurzel"?

Reply:
=> `Füge weitere Elemente des selben Konzeptes hinzu`